ct-testing/tests/example-test-ansible-setup.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
I realize these are just examples, but we should also be able to get these installed with this rpm.  I like the idea of having our own spot to work, but we'd eventually need a change to the spec file or move these in with a path that gets installed already.